README.md

```diff
@@ -112,24 +112,31 @@ source ./setup_shell.sh
 ### 3. Setting up vendors
 
 RAI is vendor-agnostic. Use the configuration in [config.toml](./config.toml) to set up your vendor of choice for RAI modules.
+You can skip the setup of optional modules for now. It is covered in the [Developer Guide](docs/developers_guide.md).
 
-If you do not have a vendor's key, follow the instructions below:
+| Module                     | Open source | Alternative             | Why to consider alternative?                                           | More information                                                                                                              |
+| -------------------------- | ----------- | ----------------------- | ---------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| AI service                 | Ollama      | OpenAI, Bedrock         | Overall performance of the AI model, supported modalities and features | [LangChain models](https://docs.langchain4j.dev/integrations/language-models/)                                                |
+| _Optional:_ Tracing tool   | Langfuse    | LangSmith               | Better integration with LangChain                                      | [Comparison](https://langfuse.com/faq/all/langsmith-alternative)                                                              |
+| _Optional:_ Text to speech | OpenTTS     | ElevenLabs              | Arguably, significantly better voice synthesis                         | <li> [OpenTTS GitHub](https://github.com/synesthesiam/opentts) </li><li> [RAI voice interface](docs/voice_interface.md) </li> |
+| _Optional:_ Speech to text | Whisper     | OpenAI Whisper (hosted) | Larger (stronger) models are available                                 | <li> [Whisper GitHub](https://github.com/openai/whisper) </li><li> [RAI voice interface](docs/voice_interface.md) </li>       |
```

Comment:
I think this table should be outside of the main readme. It's getting cluttered.
In terms of Whisper, I am not even sure if the hosted version offers bigger models. I actually don't think so. They offer less, but sometimes there is no choice if there is no gpu onboard for asr